packages/core/src/internal/decorators/element.ts

```diff
@@ -1,10 +1,10 @@
 /*
- * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { registerElementSafely } from './../utils/register.js';
+import { registerElementSafely } from './../utils/registration.js';
```

Comment:
Like `index`, `register` is the convention for a side effects. Was easier to rename the file than to come up with clever glob/regex matching